generators/profilegen/src/resources/templates/rollup/client_template.template

```diff
@@ -2,45 +2,23 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 
-<%- if @individual_gem_profile -%>
-require 'profiles/<%= @profile_name.downcase %>/<%= @module_definition_file_name.gsub('.rb','') %>'
-<%- @file_names.each do |file_name| -%>
-require 'profiles/<%= @profile_name.downcase %>/modules/<%= file_name %>'
-<%- end -%>
-require 'profiles/common/configurable'
-require 'profiles/common/default'
-<%- else -%>
 require '<%= @profile_name.downcase %>/<%= @module_definition_file_name.gsub('.rb','') %>'
 <%- @file_names.each do |file_name| -%>
 require '<%= @profile_name.downcase %>/modules/<%= file_name %>'
 <%- end -%>
 require 'common/configurable'
 require 'common/default'
-<%- end -%>
 
-<%- if @individual_gem_profile -%>
-module Azure::<%= @class_name %>::Profiles::<%= @profile_name %>::Mgmt
-<%- else -%>
 module Azure::Profiles::<%= @profile_name %>::Mgmt
-<%- end -%>
   #
   # Client class for the <%= @profile_name %> profile SDK.
   #
-  <%- if @individual_gem_profile -%>
-  class Client < <%= "#{@class_names[0]}" %>Class
-  <%- else -%>
   class Client
-  <%- end -%>
     include Azure::ARM::Configurable
 
-    <%- if !@individual_gem_profile -%>
     attr_reader <% @class_names.each_with_index do |class_name, index| %> :<%= class_name.gsub(/([a-z\d])([A-Z])/,'\1_\2').downcase %><% if index != @class_names.length - 1 %>,<% end %><% end %>
```

Comment:
similarly here, could we have less logic in the template and more in the code behind?